sechub-pds-solutions/loc/docker/scripts/scc_mock.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+# SPDX-License-Identifier: MIT
+
+echo "Running PDS Scc Mock"
+cp "$MOCK_FOLDER/scc-mock.json" "$PDS_JOB_RESULT_FILE"
```

Comment:
Please add as a comment, what repository you used to generate the mock file output. We might have to adjust that if `scc` changes in the future (have a look at the `cloc` mock file for an example.

Reply:
ok. done